users/service.go

```diff
@@ -133,16 +133,17 @@ func (svc service) ViewClient(ctx context.Context, token, id string) (mgclients.
 		return mgclients.Client{}, err
 	}
 
+	client, err := svc.clients.RetrieveByID(ctx, id)
```

Comment:
We should not be able to see users' metadata and tags when listing users. I.e. data when fetching a single user can only be a superset of data returned by fetching users during listing.